frontend/src/components/Collections/components/Grid/components/Row/DatasetRow/components/MoreDropdown/components/Menu/index.tsx

```diff
@@ -0,0 +1,40 @@
+import {
```

Comment:
`<DatasetRow />` has its `<MoreDropdown />` component, and this `<Menu />` is the content we pass to `<MoreDropdown />`